README.md

```diff
@@ -40,11 +41,83 @@ UI to view the detailed approval status of any pull request.
 - [Contributing](#contributing)
 - [License](#license)
 
+## Designing Policies
+
+An approval **policy** is a set of **rules** that combine to determine when a
+pull request is approved. Policy Bot reports approval as a successful status
+check on the head commit of the pull request. The most common form of approval
+is code review from other developers, but Policy Bot supports many other types
+of approval.
+
+Rules combine using the logical operators `and` and `or`. With no operators, a
+policy requires that all of its rules are approved to approve the pull request.
+
+### Rules
+
+Rules define the specific circumstances in which a pull request is approved.
+Each rule has four components:
+
+1. **Name**. This is an arbitrary string that you use to refer to the rule in
+   a policy. It also appears in the details view for a pull request. Try to
+   pick names that make sense to humans and explain the purpose of the rule. A
+   good name can complete the sentence "This pull request is approved if ..."
+
+2. **Predicates**. A rule is only active when all of its predicates
+   are true. If any predicate is false, the rule is skipped and does not
+   influence the policy. Predicates allow you to create rules that are
+   conditional on the state of the pull request, like requiring approval from a
+   special team when specific important files change.
+
+3. **Options**. Options control the behavior of the rule. This includes things
+   like the approval methods, if pushing new changes invalidates previous
+   approvals, and if the rule should request reviews from users or teams.
+
+4. **Requirements**. These are the things that must be true for the rule to be
+   approved. Requirements can be approvals from users with the right properties
+   or can be conditions about the pull request, similar to predicates. If a
+   rule has no requirements, it is always approved.
+
+Only a name is required, but most rules need to include either predicates or
+requirements to be useful.
+
+After evaluation, a rule can be in one of four states:
+
+* `approved` - all of the predicates and requirements are true
+* `pending` - all of the predicates are true but one or more requirements is not true
+* `skipped` - one or more predicates are not true
+* `error` - something went wrong while evaluating the rule
+
+For the purposes of the `and` and `or` operators:
+
+* `approved` is equivalent to `true`
+* `pending` and `error` are equivalent to `false`
+* `skipped` completely removes the rule from the condition
+
+### Predicates vs Conditions
```

Comment:
Did you mean `Requirements` instead of `Conditions` here?

Reply:
Updated to use "required conditions" - "requirements" includes both normal review/approval from users and the new conditions feature, so I want to distinguish between them.